core/metrics/otlp.go

```diff
@@ -101,6 +125,17 @@ const (
 	otlpTransportGRPC                              // grpc
 )
 
+func buildClientFromTransport(transport otlpTransportType) (otlptrace.Client, error) {
```

Comment:
Relevant docs:

[exporters general doc](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#otlp-exporter)
[exporter envs](https://opentelemetry.io/docs/specs/otel/protocol/exporter/) these are used for construction of otlp client

sent a loom but tl;dr, at the moment you don't actually allow different urls b/c any otlptracehttp.NewClient(), or otlptracehttp.NewClient(), is going to use the defualt enviornment variable name

Seperately, make sure you add new enviornment variables to metrics/README.md going forard